_DeploymentAndDistroPackaging/ansible/group_vars/all

```diff
@@ -1,4 +1,5 @@
 ---
+workdir: /tmp/ciao
```

Comment:
What are the permissions on the ciao directory if it is created?  What happens if it is already present and globally readable before the start of the ansible script?

Do you leak the private certificates to other users on the deployment machine?